def/taskUtils.go

```diff
@@ -50,7 +52,8 @@ func (tc TasksToClassify) taskObservationCalculator(task Task) []float64 {
 	} else if task.Watts != 0.0 {
 		return []float64{task.Watts}
 	} else {
-		log.Fatal("Unable to classify tasks. Missing Watts or ClassToWatts attribute in workload.")
+		elektronLogging.ElektronLog.Log(elekLogT.ERROR, log.FatalLevel,
```

Comment:
This is very confusing. In fact, now that we are using logrus, we can get rid of the following types in `types.go`.
1. ERROR
2. GENERAL
3. FATAL
4. WARNING

Instead, we should just mention that we are logging to the console. The log level is anyway specified as the second argument.

Reply:
Yeah, that's right.